config.yml

```diff
@@ -7723,3 +8036,2703 @@ production_recipes:
+    inputs:
+      Fishing Rod:
+        material: FISHING_ROD
+        amount: 1
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 1
+    outputs:
+      Enchanted Fishing Rod:
+        material: FISHING_ROD
+        amount: 1
+    enchantments:
+      Unbreaking 3:
+        type: DURABILITY
+        level: 3
+        probability: 0.16
+      Efficiency 4:
+        type: DIG_SPEED
+        level: 4
+        probability: 0.16
+      Efficiency 4:
+        type: DIG_SPEED
+        level: 4
+        probability: 0.06
+      Efficiency 4:
+        type: DIG_SPEED
+        level: 4
+        probability: 0.05
+  Create_Bow:
+    name: Create Bow
+    production_time: 40
+    inputs:
+      Bow:
+        material: BOW
+        amount: 1
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 2
+    outputs:
+      Enchanted Bow:
+        material: BOW
+        amount: 1
+    enchantments:
+      Unbreaking 3:
+        type: DURABILITY
+        level: 3
+        probability: 0.16
+      Power 4:
+        type: ARROW_DAMAGE 
+        level: 4
+        probability: 0.26
+      Power 3:
+        type: ARROW_DAMAGE 
+        level: 3
+        probability: 0.18
+      Flame 1:
+        type: ARROW_FIRE 
+        level: 1
+        probability: 0.12
+      Punch 1:
+        type: ARROW_KNOCKBACK
+        level: 1
+        probability: 0.07
+      Infinity 1:
+        type: ARROW_INFINITE 
+        level: 1
+        probability: 0.06
+      Punch 2:
+        type: ARROW_KNOCKBACK
+        level: 2
+        probability: 0.05
+  Create_Diamond_Sword:
+    name: Create Diamond Sword
+    production_time: 40
+    inputs:
+      Diamond Sword:
+        material: DIAMOND_SWORD
+        amount: 1
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 2
+    outputs:
+      Enchanted Diamond Sword:
+        material: DIAMOND_SWORD
+        amount: 1
+    enchantments:
+      Unbreaking 3:
+        type: DURABILITY
+        level: 3
+        probability: 0.27
+      Knockback 2:
+        type: KNOCKBACK
+        level: 2
+        probability: 0.26
+      Sharpness 3:
+        type: DAMAGE_ALL
+        level: 3
+        probability: 0.04
+      Smite 4:
+        type: DAMAGE_UNDEAD
+        level: 4
+        probability: 0.17
+      Bane of Arthropods 4:
+        type: DAMAGE_ARTHROPODS
+        level: 4
+        probability: 0.16
+      Sharpness 4:
+        type: DAMAGE_ALL
+        level: 4
+        probability: 0.03
+      Fire Aspect 2:
+        type: FIRE_ASPECT
+        level: 2
+        probability: 0.02
+      Looting 3:
+        type: LOOT_BONUS_MOBS
+        level: 3
+        probability: 0.0115
+      Looting 2:
+        type: LOOT_BONUS_MOBS
+        level: 2
+        probability: 0.007
+      Smite 5:
+        type: DAMAGE_UNDEAD
+        level: 5
+        probability: 0.012
+      Bane of Arthropods 4:
+        type: DAMAGE_ARTHROPODS
+        level: 4
+        probability: 0.0105

```

Comment:
Bane lvl 4 appears twice, given probability I'm guessing this was meant to be Bane 5.

Note that my actual realized presence of B5 in normal enchanting was closer to 0.04, but my sample size was only maybe a hundred swords.